lib/utils.pm

```diff
@@ -66,6 +66,7 @@ our @EXPORT = qw(
   service_action
   assert_gui_app
   install_all_from_repo
+  slow_run_scripted_commnad
```

Comment:
s/comnad/command/, or maybe even better name: `run_scripted_command_slow` to sound similar to `type_string_slow`